lib/packages/core/src/api/__tests__/transactionApi.spec.ts

```diff
@@ -70,7 +71,7 @@ describe('Transaction Api', () => {
             httpMock = HttpMockBuilder.create()
             // tslint:disable:max-line-length
             .onPostReply(200, mockBroadcastResponse,
-                'relPath?requestType=sendMoney&requestType=sendMoney&amountNQT=100000000&publicKey=recipientId&recipient=senderPrivateKey&deadline=1440&feeNQT=100000000')
+                'relPath?requestType=sendMoney&amountNQT=100000000&publicKey=senderPublicKey&recipient=recipientId&deadline=1440&feeNQT=100000000')
```

Comment:
Fixed a test error that mightve confused people

Reply:
good catch